recipe/testdata/file.notzip

```diff
@@ -0,0 +1 @@
+This is definitely not a zip.
```

Comment:
Is it required to add another `test` directory? Maybe we could reuse the existing one, or move the content of the existing one (`test`) to the new one (`testdata`) and get rid of the old one?

Reply:
We are using testdata directory because it is ignored by the go tool. 
[Test packages](https://golang.org/cmd/go/#hdr-Test_packages):
`The go tool will ignore a directory named "testdata", making it available to hold ancillary data needed by the tests.`

Therefore I think it will be better to move the files from test directory to testdata.

As far as I see test/site.go file is not used in any test. 
Also test/go tar file is used in recipe\_test.go to test the Unzip method. Since we now have separate test for unzip, maybe we can remove the test/go and test/site.go files as well as recipe\_test.go?